control_panel_api/serializers.py

```diff
@@ -16,7 +16,7 @@ class Meta:
         fields = ('id', 'url', 'name')
 
 
-class ProjectSerializer(serializers.HyperlinkedModelSerializer):
+class AppSerializer(serializers.HyperlinkedModelSerializer):
     class Meta:
-        model = Project
-        fields = ('id', 'url', 'name', 'slug', 'repository')
+        model = App
+        fields = ('id', 'name', 'slug', 'repo_url')
```

Comment:
We should keep the `url` as well here as in the other views. (The `url` in this case is the "API hyperlink" which is kind of nice).